rpm/ssu.spec

```diff
@@ -21,7 +21,6 @@ Requires(pre): shadow-utils
 Requires(pre): /usr/bin/groupadd-user
 Requires(postun): shadow-utils
 Requires: ssu-vendor-data
-Requires: ssu-network-proxy
```

Comment:
Some Obsoletes/Provides: here to get the package nicely uninstalled?

Reply:
Fixed. Not sure whether there are alternatives to the `network-proxy-plugin` package, providing `ssu-network-proxy`.